index.html

```diff
@@ -330,13 +364,22 @@ <h5 class="modal-title">Registration successful</h5>
     });
 };
 
-window.EBWidgets.createWidget({
-    widgetType: 'checkout',
-    eventId: '45044537424',
-    modal: true,
-    modalTriggerElementId: 'eventbrite-widget-modal-trigger-45044537424',
-    onOrderComplete: checkoutCallback
-});
+
+document.getElementById("register").onclick = function(e) {
+  $('#registrationModal').modal({show: true,
+  keyboard: false,
+  backdrop: 'static'});
+  e.preventDefault();
```

Comment:
So when/how does our form record the user's input? I wonder if it would be more foolproof if we records the user's input before we pop up the iframe. We will need to manually double check to see if everyone fill out the form pays, but that shouldn't be too hard.

Reply:
We can send the data to the server when they click `proceed to payment` and then send a short confirmation message when they close the modal. The email from the cvent website will act as the real proof of registration.